src/main/java/com/shigotosagashi/jobdomainservice/controller/JobController.java

```diff
@@ -20,30 +21,31 @@
 public class JobController {
 
   @Autowired
-  private ExampleEntityRepository exampleEntityRepository;
+  private JobEntityRepository jobEntityRepository;
 
   @PostMapping
-  public ExampleEntityDto save(
-      @Valid @RequestBody com.shigotosagashi.jobdomainservice.dto.in.ExampleEntityDto example) {
-    return ExampleEntityDto.fromEntity(exampleEntityRepository.save(example.toEntity()));
+  public JobOut save(
```

Comment:
NABD, but you could've saved a couple of lines in these methods. There's no need to write the parameters in new lines.

```
public JobOut save(@Valid @RequestBody JobIn example)
public JobOut update(@PathVariable String id, @Valid @RequestBody JobIn example)...
```